### ISSUE TITLE
Test summary not shown in report

### ISSUE THREAD

**Issue by pjan** [disneystreaming/weaver-test#25]
Per the examples shown in the [docs](https://disneystreaming.github.io/weaver-test/docs/multiple_suites_success), I don't get the summary output at the end of running tests.

Expected: `Total 2, Failed 0, Passed 0, Ignored 0, Cancelled 0`
Actual: (nothing)

Using the latest version of mill, scala 2.13 and weaver.

**Comment by Baccata** [comment]
Hi @pjan, can you provide a minimal reproducible example (in the form of a git repo for example) so that we can have a look ?

**Comment by Baccata** [comment]
Sorry, I had slightly misunderstood the problem. 

These docs you refer to are generated using our [custom runner](https://github.com/disneystreaming/weaver-test/blob/master/modules/core/src/weaver/Runner.scala) which emulates what the build tools do when calling upon a test framework (we implemented it with SBT output in mind). 

When using a test framework (as in, one that fits in the current status-quo by implementing [sbt's test interface]) in the context of a build tool, it is the responsibility of the build tool, not the one test framework, to decide whether it should create that test summary. mill does not do it (I've just tried with utest). 

Therefore, in our opinion, this issue should be raised with mill.
